x-pack/test/api_integration/apis/xpack_main/telemetry/telemetry_local.js

```diff
@@ -313,10 +314,38 @@ export default function ({ getService }) {
         'stack_stats.xpack.watcher.enabled',
         'stack_stats.xpack.watcher.execution.actions._all.total',
         'stack_stats.xpack.watcher.execution.actions._all.total_time_in_ms',
+        'stack_stats.xpack.watcher.watch.action._all.active',
```

Comment:
__note:__ I'm a bit concerned that so many new fields appeared after changing `discovery.zen.ping.unicast.hosts` to `[]` and `cluster.initial_master_nodes` to `127.0.0.1`. :man_shrugging: @tsullivan does it ring any bell?